mne/datasets/epilepsy_ecog/_data.py

```diff
@@ -15,9 +17,20 @@
 def data_path(
         path=None, force_update=False, update_path=True,
         download=True, verbose=None):  # noqa: D103
-    return _data_path(path=path, force_update=force_update,
-                      update_path=update_path, name='epilepsy_ecog',
-                      download=download)
+    # import pooch library for handling the dataset downloading
+    pooch = _soft_import('pooch', 'dataset downloading', strict=True)
+    dataset_params = {'epilepsy_ecog': epilepsy_ecog}
+    config_key = epilepsy_ecog['config_key']
+
+    # get download path for specific dataset
+    path = _get_path(path=path, key=config_key, name='epilepsy_ecog')
+
+    # instantiate processor that unzips file
+    processor = pooch.Untar(extract_dir=path)
+
+    return fetch_dataset(dataset_params=dataset_params, processor=processor,
+                         path=path, force_update=force_update,
+                         update_path=update_path, download=download)
```

Comment:
such blocks of code seem to be a lot of code duplication. Can you avoid it with a private function?